ci/tests/richards.c

```diff
@@ -1,6 +1,6 @@
 /*  C version of the systems programming language benchmark
-**  The original benchmark code is in public domain. Accordingly 
-**  all changes made by J.Singer to this benchmark is also released 
+**  The original benchmark code is in the public domain. Accordingly 
+**  all changes made by J.Singer to this benchmark are also released 
```

Comment:
s/J.Singer/J. Singer/

Reply:
Done in [89c86f9](https://github.com/capablevms/bdwgc/pull/18/commits/89c86f9a5d2e84155909d89269561b7359026a37)